sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -549,8 +554,11 @@ public BlobProperties downloadToFile(String filePath, boolean overwrite) {
         ParallelTransferOptions parallelTransferOptions, DownloadRetryOptions downloadRetryOptions,
         BlobRequestConditions requestConditions, boolean rangeGetContentMd5, Set<OpenOption> openOptions,
         Duration timeout, Context context) {
+        Context updatedContext = context == null ? Context.NONE.addData(CoreConstants.DISABLE_BUFFER_COPY, true)
+            : context.addData("disable-buffer-copy", true);
```

Comment:
nit; Context.NONE.addData(CoreConstants.DISABLE_BUFFER_COPY, true)

Reply:
Do we need to use `Context.NONE` in this situation? Instead could a new `Context` be used?

Reply:
Context doesn't have a simple helper method to create one with just 1 key-value pair. I had to create a map first and then create context, instead, I used this alternative which was simpler.